crud/count.go

```diff
@@ -88,27 +88,27 @@ func NewCountRequest(space string) *CountRequest {
 
 // Conditions sets the conditions for the CountRequest request.
 // Note: default value is nil.
-func (req *CountRequest) Conditions(conditions []Condition) *CountRequest {
+func (req CountRequest) Conditions(conditions []Condition) CountRequest {
```

Comment:
Does such setters allowed in go?

Reply:
Yea, there is a difference. A usage pattern should be:
```Go
mys2 = mys2.A(4)
```
You could see the pattern wih `SomeRequest` types in tests and examples.

> And from the first glance, it seems that it is some S3 with different handles.

Yes, there will be a difference with the main package, but it is highlighted by `Make` naming. I plan to make requests in the main package immutable too in `v2`.

> The breaking change is a separate question -- it seems that we rely on nobody using crud from master now

I think before a release we are free to change the behavior.

Reply:
> I think before a release we are free to change the behavior.

Does specifying `https://github.com/tarantool/go-tarantool` fetches last tag or current master?

Reply:
Yes, it will be a latest release version by default. You need to specify a latest commit manually for `master`.